frontend/src/components/ui/BuyMenu.jsx

```diff
@@ -72,6 +72,7 @@ export default function BuyMenu({game}) {
     useEffect(() => {
         // Handle game state changes
         deselectAll();
+    //eslint-disable-next-line
```

Comment:
A bit broad, could use this to make more specific:

```suggestion
    //eslint-disable-next-line react-hooks/exhaustive-deps
```


... or instead add this to `.eslintrc.cjs`:

```
'react-hooks/exhaustive-deps': 'off', // suppress useEffect dependency warnings
```